tcl/auth.tcl

```diff
@@ -76,7 +76,7 @@ proc qc::auth_session { session_id } {
     #| Try to authenticate user based on the session_id given
     #| Return the user_id if successful
     #| On failure throw AUTH error
-    if { [session_exists $session_id] } {
+    if { [session_valid $session_id] } {
         return [session_user_id $session_id]
```

Comment:
Can the definition of age and idle_timeout be config parameters with defaults if not defined.